packages/process/src/node/process.ts

```diff
@@ -61,7 +61,7 @@ export abstract class Process {
     protected handleOnExit(event: IProcessExitEvent) {
         this._killed = true;
         const signalSuffix = event.signal ? `, signal: ${event.signal}` : '';
-        this.logger.info(`Process ${this.pid} has exited with code ${event.code}${signalSuffix}.`);
+        this.logger.debug(`Process ${this.pid} has exited with code ${event.code}${signalSuffix}.`);
```

Comment:
if code is not zero then it should be at least warning

Reply:
Ok, we can do that, but then we would need to improve the message to include the process name, and maybe the arguments.  Otherwise, you just know that a process exited with exit code 1, for example, but you have no idea what the process was.

Reply:
sounds good